specification/network/resource-manager/Microsoft.Network/preview/2021-05-01-preview/networkManager.json

```diff
@@ -507,6 +507,14 @@
                 "type": "string"
               },
               "description": "List of subscriptions."
+            },
+
+            "crossTenantScopes": {
```

Comment:
Everything about cross tenant scopes inside the nm model needs to be read-only.

Reply:
By adding readOnly here, the list is then readOnly as well, right? Or do I need to add the readonly attribute to the CrossTenantScopes properties as well?